tests/unit/certGenInitFunction.unitTest.ts

```diff
@@ -17,8 +17,8 @@ describe("certGenInit Function", () => {
         const result = await certGenInit(undefined, ctx, () => {
```

Comment:
since the handler is async, i think the properway to test it is await(expect(certgeninit(undefined,..... => {})).rejects.toThrow('ur error message")

Reply:
Agree, this is a really old jest pattern that has been massively improved upon. Again falls into the this lambda is not long for this world camp and I don't think refactoring all of this is worth the time investment.